docs/scripts_md/Notify.md

```diff
@@ -0,0 +1,113 @@
+# NAME
+
+NeuroDB::Notify -- Provides an interface to the email notification subsystem
+of LORIS
+
+# SYNOPSIS
+
+    use NeuroDB::Notify;
+
+    my $notifier = NeuroDB::Notify->new(\$dbh);
+
+    my $message           = "Some kind of message from tarchive validation";
+    my $upload_id         = 123456;
+    my $notify_notsummary = 'N';
+    $notifier->spool('tarchive validation', $message,   0,
+                             'tarchiveLoader',      $upload_id, 'Y',
+                             $notify_notsummary
+                            );
+
+# DESCRIPTION
+
+This class defines an interface into the email notification subsystem of
+LORIS - particularly with regards to spooling new messages.
+
+## Methods
+
+### new($dbh) >> (constructor)
+
+Creates a new instance of this class. The parameter `\$dbh` is a
+reference to a DBI database handle, used to set the object's database
+handle, so that all the DB-driven methods will work.
+
+INPUT: DBI database handle
+
+RETURNS: new instance of this class.
+
+### spool($type, $message, $centerID, $origin, $processID, ...)
+
+Spools a new notification message, `$message`, into the `notification_spool`
+table for notification type `$type`. If `$centerID` is specified, only
+recipients in that site will receive the message.
+
+INPUTS:
+  - $type     : notification type
+  - $message  : notification message
+  - $centerID : center ID
+  - $origin   : notification origin
+  - $processID: process ID
+  - $isError  : if the notification is an error
+  - $isVerb   : if verbose is set
+
+RETURNS: 1 on success, 0 on failure
+
+### getTypeID($type)
+
+Gets the notification typeID for the notification of type `$type`.
+
+INPUT: notification type
+
+RETURNS: the notification typeID, or undef is none exists
```

Comment:
a small typo I missed the first time;
undef IF not undef IS

Reply:
wow! good catch, I had trouble seeing it ;)